docs/ContinuousIntegration.md

```diff
@@ -0,0 +1,117 @@
+# Firebase Apple SDK Continuous Integration
+
+Firebase uses [several GitHub Action workflows](.github/actions/) for product testing. Most of the
```

Comment:
This link and the links below don't work - links should be relative to this doc, not repo relative.